### DISCUSSION TITLE
Features wanted

### DISCUSSION THREAD

**Discussion by Draecal** [jesec/flood#389] [Ideas]
- Select files to download when .torrent has multiple files inside.
- Add % value to the progress bar.
- Select not to create a folder when downloading .torrent files.
- Rename files inside .torrent.

**Comment by jesec** [comment]
> Select files to download when .torrent has multiple files inside.

You can do that in the files tab of details panel. 

> Add % value to the progress bar.

That would not fit well with the design, and I don't think it would be useful.

> Select not to create a folder when downloading .torrent files.

"Use as Base Path".

> Rename files inside .torrent.

No. That proves to be inconsistent and error-prune. Don't do that. And, not all torrent clients support that.

**Reply by Draecal** [reply]
> > Select files to download when .torrent has multiple files inside.
> 
> You can do that in the files tab of details panel.
> 
Is that available when you open multiple .torrent files at once?
